src/main/smithy/sonos.smithy

```diff
@@ -0,0 +1,63 @@
+namespace com.kubukoz.next.sonos
+
+use smithy4s.api#simpleRestJson
+
+@simpleRestJson
+service SonosApi {
+  version: "0.0.0",
+  operations: [NextTrack, Seek, GetZones]
+}
+
+@http(method: "POST", uri: "/{room}/next")
+operation NextTrack {
+  input: NextTrackInput,
+}
+
+structure NextTrackInput {
+  @required
+  @httpLabel
+  room: String
+}
+
+
+@http(method: "GET", uri: "/{room}/timeseek/{seconds}")
```

Comment:
This currently fails to generate and can't be suppressed AFAIK (error level notice from java smithy API), but I don't think it's accurate.

The planned workaround that I could use is hardcoding this to my actual room name because I only have one and probably nobody else is using this :joy:

Reply:
@baccata I suppose this needs to be fixed in awslabs/smithy?

Reply:
Yes it does

Reply:
https://github.com/awslabs/smithy/issues/1029

Also, as another workaround you could create 2 different services to hold the conflicting operations ?

Reply:
Yeah, most likely